src/ts/src/common/components/library/button/index.module.scss

```diff
@@ -0,0 +1,45 @@
+@import "src/common/styles/spacing";
+@import "src/common/styles/typography";
+@import "src/common/styles/colors";
+@import "src/common/styles/corners";
+
+.button {
+  width: 7.5em;
+  height: 2.125em;
+  border-radius: $border-radius-l;
+  border-style: solid;
+  border-color: $primary;
+  border-width: 0.1em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  &:hover {
+    background-color: $primary-dark;
+    border-color: $primary-dark;
+    color: $white;
+    cursor: pointer;
+  }
+
+  &:active {
```

Comment:
can you remind me what the & sign does again?

Reply:
Great question! It's the [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector), which is a shortcut for using the parent selector for this pseudo class (`:active`)!

E.g.,

Input:
```scss
.alert {
  // The parent selector can be used to add pseudo-classes to the outer
  // selector.
  &:hover {
    font-weight: bold;
  }

  // It can also be used to style the outer selector in a certain context, such
  // as a body set to use a right-to-left language.
  [dir=rtl] & {
    margin-left: 0;
    margin-right: 10px;
  }

  // You can even use it as an argument to pseudo-class selectors.
  :not(&) {
    opacity: 0.8;
  }
}
```

Output:
```css
.alert:hover {
  font-weight: bold;
}
[dir=rtl] .alert {
  margin-left: 0;
  margin-right: 10px;
}
:not(.alert) {
  opacity: 0.8;
}

```